src/components/Logos/AWSAthena.tsx

```diff
@@ -0,0 +1,63 @@
+import { SVGAttributes } from "react";
+
+const AWSAthena = (props: SVGAttributes<SVGElement>) => (
+  <svg
+    width="64"
+    height="64"
+    viewBox="0 0 64 64"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <g clip-path="url(#clip0_5755_3307)">
+      <g clip-path="url(#clip1_5755_3307)">
+        <path
+          d="M60 4H4V60H60V4Z"
+          fill="url(#paint0_linear_5755_3307)"
+        />
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
```

Comment:
these attributes need to be camelCase otherwise we will get console errors when rendering these. for example `clicp-path` needs to be `clipPath`, `fill-rule` -> `fillRule` etc